string object_name
std_msgs/ColorRGBA color
---
string status
